multitenancy/src/test/java/org/axonframework/extensions/multitenancy/configuration/MultiTenantEventProcessingModuleTest.java

```diff
@@ -40,6 +40,7 @@
 import org.junit.jupiter.api.*;
 import org.mockito.*;
 
+import javax.sound.midi.Track;
```

Comment:
I love imported music, but it may not be useful here 😉

Reply:
Don't have a clue how that happened 😁